tests/conftest.py

```diff
@@ -4780,6 +4780,7 @@ def setup_acm_ui(request):
 
 
 def setup_acm_ui_fixture(request):
+    ocsci_config.switch_acm_ctx()
```

Comment:
@sidhant-agrawal , what happens after the test ? context will be still with ACM. I suggest you to store old context during setup and switch to old context in the finalizer so that ocs-ci works in that old context . WDYT ?